README.rst

```diff
@@ -64,6 +64,9 @@ Add the following inside Scrapy's ``settings.py`` file:
         "scrapy_poet.RetryMiddleware": 275,
     }
 
+    from web_poet import default_registry
+    SCRAPY_POET_RULES = default_registry.get_rules()
```

Comment:
That's actually a question I had in a previous PR, but haven't asked. Would it make sense to use these rules by default?

Reply:
> Would it make sense to use these rules by default?

I'm not quite sure at the moment since I'm not sure how users would use their rules. For example, they might only be picking some of them using `default_registry.search()` or directly creating the `ApplyRule` instances form here.

Reply:
It may still make sense to use the default registry rules by default. The other options are still possible by changing the value.

Reply:
I've created another PR in #120 to explore this alternative.